e2e/programmatic/get-meta.js

```diff
@@ -0,0 +1,5 @@
+import getMeta from "@ladle/react/meta";
+
+const meta = await getMeta();
+
+console.log({ meta });
```

Comment:
```suggestion
```

Reply:
Can we add a test into hello.spec.ts for

```js
import getMeta from "@ladle/react/meta";
```

It's not going to load a page or anything like that, but we can still use the playwright test runner in this workspace (that has stories).